ferrocene/ci/configure.sh

```diff
@@ -34,115 +25,45 @@ if [[ -z "${CI+x}" ]]; then
     echo
     echo "    https://public-docs.ferrocene.dev/main/qualification/internal-procedures/setup-local-env.html"
     echo
-    echo "If you're trying to simulate CI to debug an issue locally, set the"
-    echo '$CI' "environment variable when invoking this script to suppress"
-    echo "this error message."
+    echo "If you're trying to simulate CI to debug an issue locally, change the"
+    echo "profile in your config.toml to 'ferrocene-dist'."
     exit 1
 fi
 
-# Helper function to check whether we should include things that only Ferrous
-# Systems has access to in the configuration.
-is_internal() {
-    if [[ -z "${OUTSIDE_FERROUS+x}" ]]; then
-        return 0
-    else
-        return 1
-    fi
-}
+if [[ -n "${OUTSIDE_FERROUS+x}" ]]; then
```

Comment:
Is this supposed to be `FERROCENE_UNPRIVILEGED_CI`?

Reply:
No. The purpose of this block is to provide a clear error and migration path for customers building Ferrocene from source using `OUTSIDE_FERROUS=1`.